Sources/Prometheus/MetricTypes/Counter.swift

```diff
@@ -1,9 +1,11 @@
+import NIOConcurrencyHelpers
+
 /// Prometheus Counter metric
 ///
 /// See https://prometheus.io/docs/concepts/metric_types/#counter
```

Comment:
Is `- See: https://prometheus.io/docs/concepts/metric_types/#counter` better perhaps?
If yes, perhaps apply on all other See links?

Reply:
Don't think that does anything?